### PR TITLE
add experiment article

### PR DESCRIPTION
**Pull request by stephamon**
JIRA 2331

### CODE REVIEW

**Comment by stephamon** [comment]
@cyrichardson can you take a look at this please?

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-59a22fe7db/how-to/)

**Comment by cyrichardson** [comment]
On it, @stephamon.

**Comment by cyrichardson** [comment]
@stephamon Some suggestions here: 
[shut-down-a-server_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2311201/shut-down-a-server_cyrEdits.docx)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-94d9f67cde/how-to/). :bow:

File: content/cloud-servers/shut-down-a-server.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+last_modified_date: '2018-08-28'
+last_modified_by: Stephanie Fillmon
+product: Cloud Servers
+product_url: cloud-servers
+---
+
+A common use case for shutting down a server is to determine whether the
+server is still actively in use. Sometimes shutting down a server is
+the quickest and easiest way to make this determination. Shutting
+down the server will *not* end the charges for the server. As long as
```
Comment (catlook) [inline content/cloud-servers/shut-down-a-server.md]: do you want to change that "will" to "does"?

File: content/cloud-servers/shut-down-a-server.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+the server is utilizing resources, it will accrue charges.
```
Comment (catlook) [inline content/cloud-servers/shut-down-a-server.md]: and maybe "accrues" instead of "will accrue"? or "continues to accrue"?

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-b81160aa78/how-to/). :bow:

**Pull request merged**